### PR TITLE
Kamilg/update tf

### PR DESCRIPTION
**Pull request by Kamoppl**

### CODE REVIEW

File: storage_devices/disk.py [hunk 1]
```diff
@@ -288,8 +288,8 @@ def static_init(cls):
     @classmethod
     def plug_all(cls) -> Output:
         cmd = (
-            f"for i in $(find -H /sys/devices/ -path '*/scsi_host/*/scan' -type f); do echo "
-            f"'- - -' > $i; done;"
+            "find -H /sys/devices/ -path '*/scsi_host/*/scan' -type f |"
+            " xargs -P20 -I % sh -c \"echo '- - -' | tee %\""
```
Comment (Deixx) [inline storage_devices/disk.py:292]: I wonder what is the gain here - certainly not readability

File: test_utils/os_utils.py [hunk 2]
```diff
@@ -379,10 +380,13 @@ def get_udev_service_path(unit_name):
...
-    # TERM signal should be used in preference to the KILL signal, since a
-    # process may install a handler for the TERM signal in order to perform
-    # clean-up steps before terminating in an orderly fashion.
+def kill_all_io(graceful=True):
+    if graceful:
+        # TERM signal should be used in preference to the KILL signal, since a
+        # process may install a handler for the TERM signal in order to perform
+        # clean-up steps before terminating in an orderly fashion.
+        TestRun.executor.run("killall -q --signal TERM dd fio blktrace")
+        time.sleep(3)
```
Comment (Deixx) [inline test_utils/os_utils.py:390]: so... gracefulness is just trying the nice way one more time before unleashing hell? ;)

**Review by Deixx — COMMENTED** [review]
I'm not sure which part speeds up anything - the plug_all maybe?
Renaming the PR might be required

File: test_utils/os_utils.py (hunk 2 above)
Comment (Kamoppl) [reply test_utils/os_utils.py:390]: ++

File: storage_devices/disk.py (hunk 1 above)
Comment (Kamoppl) [reply storage_devices/disk.py:292]: Parallelize sata plug

**Review by katlapinka — APPROVED** [review]

**Review by Deixx — APPROVED** [review]

**Pull request merged**